<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_program"/>
   </widget>
   <addaction name="menuLanguage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_project"/>
   <addaction name="separator"/>
   <addaction name="actionEdit_project"/>
   <addaction name="separator"/>
   <addaction name="actionDelete_project"/>
  </widget>
  <action name="actionAbout_program">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/images/about.png</normaloff>:/ico/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="actionAdd_project">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/images/add.png</normaloff>:/ico/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Add project</string>
   </property>
  </action>
  <action name="actionEdit_project">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/images/edit.png</normaloff>:/ico/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit project</string>
   </property>
  </action>
  <action name="actionDelete_project">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/images/delete.png</normaloff>:/ico/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete project</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections/>
</ui>
